daemon/container.go

```diff
@@ -73,6 +73,7 @@ type CommonContainer struct {
 	MountLabel, ProcessLabel string
 	RestartCount             int
 	UpdateDns                bool
+	HasBeenStartedBefore     bool
```

Comment:
First time the Windows container OS is booted. Not related to the physical machine.